modules/gitops-verifying-argo-cd-self-healing-behavior.adoc

```diff
@@ -0,0 +1,32 @@
+[id="verifying-argo-cd-self-healing-behavior_{context}"]
+= Verifying Argo CD self-healing behavior
+
+Argo CD constantly monitors the state of deployed applications, detects differences between the specified manifests in Git and live changes in cluster, and automatically corrects them. This behavior is referred to as self-healing. 
+
+You can test and observe the self-healing behavior in Argo CD. 
+
+.Prerequisites
+
+* The sample application was deployed and configured.
+
+.Procedure
+
+. In the Argo CD dashboard, verify that your application has the `Synced` status.
+
+. Click the `app-spring-petclinic` tile in the Argo CD dashboard to view the application resources that are deployed to the cluster.
+
+. In the OpenShift Web Console, navigate to the Developer perspective.
+
+. Modify the Spring PetClinic deployment and commit the changes to the `app/` directory of the Git repository. Argo CD will automatically deploy the changes to the cluster.
+
+. Test the self-healing behavior by modifying the deployment on the cluster and scaling it up to two pods while watching the application in the OpenShift Web Console.
++
+.. Run the following command to modify the deployment:
++
+[source,terminal]
+----
+$ oc scale deployment spring-petclinic --replicas 2  -n spring-petclinic
+----
+.. In the OpenShift Web Console, notice that the deployment scales up to two pods and immediately scales down again to one pod. Argo CD detected a difference from the Git repository and auto-healed the application on the OpenShift cluster. 
+
+. In the Argo CD dashboard, click the `app-spring-petclinic` name → *App Details* → *Events*. See the event details of Argo CD detecting that the deployment resources are out of sync on the cluster and resyncing the Git repository to correct it.
```

Comment:
name/tile?
click the `app-spring-petclinic` name, and then navigate to *App Details* → *Events*. The *Events* tab displays the event details of Argo CD detecting out of sync deployment resources on the cluster and...